src/dso_api/dynamic_api/remote/clients.py

```diff
@@ -165,9 +161,11 @@ def _raise_http_error(self, response: HTTPResponse) -> None:
                 )
             else:
                 raise BadGateway(detail_message)
-        elif response.status == 403:  # "forbidden"
-            # Return 403 to client as well
-            raise NotAuthenticated(detail_message)
+        elif response.status in (401, 403):  # "unauthorized", "forbidden"
```

Comment:
no tests for this? :(

Reply:
Working on it.

Reply:
Done!

Reply:
I've have suggested using [HTTPStatus](https://docs.python.org/3.8/library/http.html?highlight=httpstatus#http.HTTPStatus) before. Tried to look it up, but I can't find anything in Github these days. Anyway, I seem to remember you didn't want to use it at the time as existing code already used literal integer values. You were though, open to using `HTTPStatus` but rather would do so in a separate PR

However here you are using integer values with their meaning in comments explained. Is that really better then just using:
```python
elif response.status in (HTTPStatus.UNAUTHORIZED, HTTPStatus.FORBIDDEN):
```
Even if all other code still uses literal integer values? I mean you have to start somewhere.

Reply:
I'm willing to switch, but then I will express a preference for rest_framework.status codes. Those have the numerical code in them, and while debugging this numerical codes are easier to look up and remember when you have to distinguish between NotAuthenticated, Unauthorized, Forbidden and PermissionDenied.

Reply:
So do the HTTPStatus emus.

Reply:
I mean in the name: rest-framework has `status.HTTP_403_FORBIDDEN`, standard lib has `HTTPStatus.FORBIDDEN`. I want to see the status code in the source code, because that's what I'm going to grep in the RFCs :)

Reply:
See updated PR.